docs/src/languages/php/_index.md

```diff
@@ -304,16 +217,224 @@ markdownify=false
+For more information about the start command, see [web commands reference](../../create-apps/app-reference.md#web-commands).
+
+By default, start commands use the PHP-CLI and run your script with `php <SCRIPT>.php`.
+Find out how and when to use [which execution mode](#execution-mode).
+
+Note that the `start` command must run in the foreground and is executed before the [deploy hook](../../create-apps/hooks/hooks-comparison.md).
+That means that PHP-FPM can't run simultaneously along with another persistent process such as [ReactPHP](https://github.com/platformsh-examples/platformsh-example-reactphp) or [Amp](https://github.com/platformsh-examples/platformsh-example-amphp).
+If you need multiple processes, they have to run in separate containers.
+
+Set up alternate start commands to:
+
+{{< codetabs >}}
+
+---
+title=Run a custom script
+file=none
+highlight=false
+---
+
+1. Add your script in a PHP file,
+2. Specify an alternative `start` command by adapting the variable:
+
+    ```yaml {location=".platform.app.yaml"}
+    web:
+        commands:
+            start: php <APP>.php
+    ```
+
+    Where `<APP>` is a file path relative to the [app root](../../create-apps/app-reference.md#root-directory).
+
+<--->
+
+---
+title=Run a custom web server
+file=none
+highlight=false
+---
+
+1. Add your web server's code in a PHP file,
+2. Specify an alternative `start` command by adapting the variable:
+
+    ```yaml {location=".platform.app.yaml"}
+    web:
+        commands:
+            start: php <APP>.php
+    ```
+
+    Where `<APP>` is a file path relative to the [app root](../../create-apps/app-reference.md#root-directory).
+3. Set the upstream to TCP:
+
+    ```yaml {location=".platform.app.yaml"}
+        upstream:
+            socket_family: tcp
+            protocol: http
+
+    ```
+
+    When you configure your app to connect via a TCP socket, the `$PORT` environment variable is automatically set.
+    See more options on [how to configure the upstream](../../create-apps/app-reference.md#upstream).
+    You might have to configure your app to connect via the `$PORT` TCP socket,
+    especially when using web servers such as [Swoole](swoole.md) or [Roadrunner](https://github.com/roadrunner-server/roadrunner).
+4. (optional) override redirects:
+
+    ```yaml {location=".platform.app.yaml"}
+    locations:
+            "/":
+                allow: false
+                passthru: true
+    ```
+
+    To let the custom web server handle redirects, the .
+
+<--->
+
+---
+title=Run specific tasks
+file=none
+highlight=false
+---
+
+To execute runtime-specific tasks (such as clearing cache) before your app starts:
+
+1. Create a separate shell script that includes all the commands to be run.
+2. Specify an alternative `start` command by adapting the following:
+
+    ```yaml {location=".platform.app.yaml"}
+    web:
+        commands:
+            start: bash <SCRIPT>.sh && php <APP>.php
```

Comment:
:hammer_and_wrench: I would assume people would generally want to start the app similarly to the default. So shouldn't this be using PHP-CGI?